modules/ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,216 @@
+// TELCODOCS-272 4.8 Using Assisted Installer to Install & Manage ocp Clusters on Bare Metal
+//
+//
+//
+= Enabling Assisted Installer Service on Bare Metal
+<<introduction,Introduction to Assisted Installer Service>>
+<<enable-assisted-installer-service,Enabling the Assisted Installer Service>>
+<<resources-explained,Resources Explained>>
+
+== Introduction
+
+The Assisted Installer Service (AIS) deploys Red Hat OpenShift Container Platform (ocp) clusters running on a Bare Metal Operator (BMO) node. Red Hat Advanced Cluster Management (RHACM) 2.3+ ships with AIS, which deploys once the `MultiClusterHub` is enabled.
+
+RHACM 2.3+ only supports *Single Node OpenShift (SNO)* cluster deployments. A SNO cluster runs OpenShift Container Platform (ocp) on top of a single BMO node. The single BMO acts as *both the supervisor and the worker node*.
+
+== Enable assisted installer service
+
+=== Prerequisites
+
+. OpenShift Container Platform 4.8+ installed on Bare Metal
+. https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.2/html/install/installing#installing-red-hat-advanced-cluster-management-from-the-operatorhub[RHACM 2.3+ with MultiClusterHub resource created]
+. Two `PersistentVolumes` (PV)
```

Comment:
s\PersistentVolume (PV)\\`PersistentVolume` CRs

Reply:
Also, what are you using the PersistentVolume storage for. Elaborate.

Reply:
Updated verbiage and researching whether PV storage is an actual prerequisite for AIS.